### PR TITLE
Tidy up

### PR DESCRIPTION
**Pull request by huoyaoyuan**
Like #15 .
Use the `new` keyword when necessary.
Avoid redundant member when interface inheriting.

### CODE REVIEW

File: osu.Framework/Graphics/Transformations/ITransform.cs [hunk 1]
```diff
@@ -10,7 +10,7 @@ namespace osu.Framework.Graphics.Transformations
     public interface ITransform : IHasLifetime
     {
         double Duration { get; }
-        bool IsAlive { get; }
+        //bool IsAlive { get; }
```
Comment (Tom94) [inline osu.Framework/Graphics/Transformations/ITransform.cs]: Why not remove the line entirely rather than commenting out?

Comment (huoyaoyuan) [inline osu.Framework/Graphics/Transformations/ITransform.cs]: @Tom94  I think if a future refactor removes this interface inheritance, a remain line can prompt that "there was a property definition here".
If you think this is redundant I will remove this line.

Comment (Tom94) [inline osu.Framework/Graphics/Transformations/ITransform.cs]: The problem I have with comments is, that if e.g. the IsAlive property gets automatically refactored, then the comment will be unaffected and confuse people. I think the compile error messages will be sufficient to create a missing property should the inheritance be removed. :)

I'd prefer the line to be removed.

Comment (huoyaoyuan) [inline osu.Framework/Graphics/Transformations/ITransform.cs]: OK. Removed now.

Comment (Tom94) [inline osu.Framework/Graphics/Transformations/ITransform.cs]: Thanks, can you update your branch such that we can merge it in?

Comment (huoyaoyuan) [inline osu.Framework/Graphics/Transformations/ITransform.cs]: @Tom94 Updated. The `new` is used now, so the only work now is to remove the redundant `IsAlive`.

**Pull request merged**